src/test/osd/CMakeLists.txt

```diff
@@ -74,6 +76,10 @@ target_link_libraries(unittest_hitset osd global ${BLKID_LIBRARIES})
 add_executable(unittest_osd_osdcap
   osdcap.cc
 )
+if(HAS_VTA)
+  set_source_files_properties(osdcap.cc PROPERTIES
+    COMPILE_FLAGS -fno-var-tracking-assignments)
+endif()
```

Comment:
@cbodley i don't understand why this is unrelated. as the commit message suggests, by disabling `var-tracking-assignments`, we can disable the warning and avoid recompiling this file.